app/templates/index.html

```diff
@@ -64,13 +64,13 @@
           items = [
```

Comment:
This might be a good opportunity to extract the `g-cloud` and `crown-hosting` items into a single template variable to avoid duplication?  e.g. then have something like `if dos_slug ... items = dos_items + base_items else items = base_items` and only have each piece of text once.